api/v2/checluster_types.go

```diff
@@ -189,6 +189,13 @@ type CheClusterDevEnvironments struct {
 	// +optional
 	// +kubebuilder:validation:Enum=Always;IfNotPresent;Never
 	ImagePullPolicy corev1.PullPolicy `json:"imagePullPolicy,omitempty"`
+	// IgnoredUnrecoverableEvents defines a list of Kubernetes event names that should
+	// be ignored when deciding to fail a DevWorkspace startup. This option should be used
```

Comment:
I might be mistaken, but I think we just refer to workspaces instead of DevWorkspaces from a Che perspective? i.e.  `when deciding to fail a DevWorkspace startup` -> `when deciding to fail a workspace that is starting`.

Same thing for `Events listed here will not trigger DevWorkspace failures.` -> `Events listed here will not trigger workspace failures.`

Reply:
I see. I fixed comments as you suggested.